distributionviewer/core/static/css/distributionviewer.styl

```diff
@@ -0,0 +1,48 @@
+$background-color = #fff;
+$normal-text = rgba(0,0,0,0.6);
+$mild-blue = #4A90E2;
+
+body {
+  background-color: $background-color;
+  color: $normal-text;
+  font-family: helvetica, sans-serif;
```

Comment:
I like the simplicity of one global stylesheet for the nature of these projects. If we get something much bigger and robust we can look at splitting up typography, palettes etc.